### PR TITLE
Allow custom boost value

### PR DESCRIPTION
**Pull request by dannydeezy**
https://linear.app/deezy/issue/ENG-183/allow-custom-boost-values

### CODE REVIEW

**Comment by dannydeezy** [comment]
- [ ] requires new version of nosft-core: https://github.com/deezy-inc/nosft-core/pull/63

File: src/components/modals/send-modal/index.js [hunk 1]
```diff
@@ -234,9 +240,20 @@ const SendModal = ({
                     onChange={feeRateOnChange}
                   />
                 </InputGroup>
+                { boostRequired ?
+                  <InputGroup className="mb-3">
+                    <Form.Label>Select an output value</Form.Label>
+                    <Form.Range
+                      min="330"
+                      max="10000"
+                      defaultValue={boostOutputValue}
```
Comment (topether21) [inline src/components/modals/send-modal/index.js:249]: Is better if you remove defaultValue and use `value={boostOutputValue}`

Comment (dannydeezy) [reply src/components/modals/send-modal/index.js:249]: thanks

**Review by topether21 — APPROVED** [review]
LGTM!

**Review by habibitcoin — APPROVED** [review]
nit: the UX/ordering is kinda weird

image

Might be better to have fee rate toggle below fee rate + output value toggle below output value

Anyhow, I'll log revisiting that in the future

**Pull request merged**